pyproject.toml

```diff
@@ -14,7 +14,7 @@ requires-python = ">=3.7"
 dependencies = [
     "tqdm",
     "numpy>=1.21",
-    "brainscore_core@git+https://github.com/brain-score/core.git@main",
+    "brainscore_core@git+https://github.com/shehadak/core.git@main",
```

Comment:
I'm guessing this is for development, let's not forget to reset

Reply:
Correct, will reset before merge